spec/system/user_uses_the_editor_spec.rb

```diff
@@ -58,7 +64,13 @@ def fill_markdown_with(content)
       article_body = find(:xpath, "//div[@id='article-body']")["innerHTML"]
       article_body.gsub!(%r{"https://res\.cloudinary\.com/.{1,}"}, "cloudinary_link")
 
-      Approvals.verify(article_body, name: "user_preview_article_body", format: :html)
+      # TODO: Convert this to an E2E test?
```

Comment:
I think we probably want to convert this into a E2E test, but I didn't want to increase the scope of this too much, so I added some tests using the `user_preview_article_body.approved.html` fixture file for now and left a note to come back to it in the future.